pages/api/exampleLib.ts

```diff
@@ -0,0 +1,11 @@
+export type Greeting = {
+  greeting: string;
+  punctuation: string;
+};
+
+export function getGreeting(): Greeting {
+  return {
+    greeting: "hello",
+    punctuation: "!",
+  };
+}
```

Comment:
May want to move this file depending on how #30 goes.

Reply:
Removed; I added a test for some of your code instead